shared/src/androidMain/kotlin/dev/johnoreilly/confetti/test/screenshot/SnapshotTransformer.kt

```diff
@@ -0,0 +1,14 @@
+package dev.johnoreilly.confetti.test.screenshot
+
+import android.graphics.Bitmap
+import androidx.compose.ui.test.SemanticsNodeInteraction
+
+fun interface SnapshotTransformer {
```

Comment:
I really just moved this, but I didn't want to break the wear tests yet, so will change that